### ISSUE TITLE
Cannot install sqllitebrowser

### ISSUE THREAD

**Issue by davidcheng220** [sqlitebrowser/sqlitebrowser#2346]
Details for the issue
--------------------
“DB Browser for SQLite” can’t be opened because Apple cannot check it for malicious software.
#### What did you do?
Install DB Browser for SQLite form safari and brew, but still get the same massage

#### What did you see instead?

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [ ] Windows: ( _version:_ ___ )
- [ ] Linux: ( _distro:_ ___ )
- [x] macOS: ( _version:10.15.5 )
- [ ] Other: ___

#### What is your DB4S version?

- [x] 3.12.0
- [ ] 3.11.x
- [ ] Other: ___

#### Did you also

- [x] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [ ] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by lucydodo** [comment]
right-click(or control key[ctrl | ⌃) on DB4S to open the context menu and select 'Open'.

**Comment by justinclift** [comment]
@davidcheng220 We've just released version 3.12.1, which is Notarised on macOS, so should (in theory) launch properly without any warnings:

&nbsp; &nbsp; https://sqlitebrowser.org/dl/#macos
